drivers/dma/dma_mcux_edma.c

```diff
@@ -642,9 +653,14 @@ static int dma_mcux_edma_suspend(const struct device *dev, uint32_t channel)
 {
 	struct call_back *data = DEV_CHANNEL_DATA(dev, channel);
 
+	if (data->transferConfig.majorLoopCounts < 2) {
+		return -ENOSYS;
+	}
```

Comment:
i dont understand this check and error code, maybe you can put a comment

Reply:
A comment will be added on the next push.